src/read_write/ply.rs

```diff
@@ -485,7 +489,7 @@ impl NodeWriter<PointsBatch> for PlyNodeWriter {
         }
 
         for (i, pos) in p.position.iter().enumerate() {
-            pos.write_le(&mut self.writer)?;
+            pos.write_le_encoded(&self.encoding, &mut self.writer)?;
```

Comment:
Isn't it a bit nonsensical to write data as anything but LE? Third-party tools won't be able to read that, and I'm not aware of a use-case for having our own compressed PLY format.

Reply:
It is still LE and third party tools can still read it, it's just in other coordinates. I here preferred interface consistency, but we can also fail the writing in case an encoding other than Plain was specified... but then again, the interface doesn't make so much sense to me...

Reply:
Good point, you're right that it makes sense to combine the ScaledToCube with PLY. I think should we add more encoding options in the future, we can return a Result<Self> from the `NodeWriter`'s `from()` function that may return `UnsupportedEncoding` or similar.

Reply:
Yes, this may be an option.
Furthermore, we could also think of including the encoding in ply comments, so we wouldn't need metadata along with the ply... what do you think?